README.md

```diff
@@ -687,12 +688,15 @@ fields in the `spec` field of the `ServiceInstance` or `ServiceBinding` resource
   in the case of the `spec` field being specified as `YAML`. Any valid `YAML` or
   `JSON` constructs are supported. Only one parameter field may be specified per
   `spec`.
-- `parametersFrom`: can be used to specify which secret, and key in that secret,
-  which contains a `string` that represents the JSON to include in the set of
-  parameters to be sent to the broker. The `parametersFrom` field is a list that
-  supports multiple sources referenced per `spec`.
+- `parametersFrom`: enables you to specify one or more secrets, and the corresponding keys within those secrets, holding JSON-formatted parameters to be sent to the 
+   broker. The `parametersFrom` field is a list that
+   supports multiple sources referenced per `spec`, defining an asymmmetric relationship where the `ServiceInstance` resource can define a number of related secrets.
+- `watchParametersFromChanges`: (boolean) This field determines whether changes to the secret values referenced in `parametersFrom` should trigger an automatic update of the service instance. This enables the service instance to reconcile and react to changes in its referenced secrets. When set to true, any change to the referenced secret values will cause the service instance to be updated. Defaults to `false`.
```

Comment:
not clear. and why are we mentioning "spec" here...?

"The parametersFrom field is a list that allows referencing multiple sources for each ServiceInstance, enabling the ServiceInstance resource to specify a collection of associated secrets or keys."

Reply:
I may not understand the first part of your suggestion. This field is a list that enables referencing multiple sources. Sources of what?  And why is specifying a collection of secrets and keys important? I think we should state the benefit here. 
"Spec" was taken from the format you suggested in the explanation. I can remove it.